pkg/scheduler/internal/queue/scheduling_queue_test.go

```diff
@@ -1506,6 +1506,124 @@ func TestPriorityQueue_MoveAllToActiveOrBackoffQueue(t *testing.T) {
 	}
 }
 
+// This unit test can only exist when SchedulerQueueingHints feature gate was dsiabled by default.
+// TODO: it will be removed after SchedulingQueueHint feature gate is enabled by default.
```

Comment:
I don't think we need a TODO comment. Also, we won't remove this test unless we remove the feature gate itself (= QHint goes to GA.) so that we can make sure all things in the scheduling queue work when people disable the feature gate.

```suggestion
// TestPriorityQueue_MoveAllToActiveOrBackoffQueueWithOutQueueingHint makes sure MoveAllToActiveOrBackoffQueue works expectedly with QueueingHint disabled.
```